### PR TITLE
Describe how to update the reward points balance

### PR DESCRIPTION
**Pull request by ZhovnirMaria**
## Purpose of this pull request

This pull request describes how to update the reward points balance from the admin panel

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [ ] Magento Open Source only
- [x] Magento Commerce only

Is this update specific to an installed feature extension

- [ ] B2B extension
- [ ] Other feature set, please specify:

## Affected documentation pages

https://docs.magento.com/user-guide/customers/account-dashboard-reward-points.html?itm_source=merchdocs&itm_medium=search_page&itm_campaign=federated_search&itm_term=Reward%20Points

### CODE REVIEW

File: src/customers/account-dashboard-reward-points.md [hunk 1]
```diff
@@ -7,3 +7,43 @@ The [Reward Points]({% link marketing/rewards-loyalty.md %}) section of the cust
...
+
+## Update Reward Points Balance
+
+The rewards points balance can be updated from the admin panel.
+
+1. On the _Admin_ sidebar, go to **Customers** > _All Customers_.
+
+1. Find the customer in the grid and click **Edit** in the _Action_ column.
+
+1. In the **Customer Information** panel, choose the **Reward Points** section.
```
Comment (leslietilling) [inline src/customers/account-dashboard-reward-points.md:19]:
```suggestion
1. Under _Customer Information_, choose the **Reward Points** section.
```

File: src/customers/account-dashboard-reward-points.md [hunk 2]
```diff
@@ -7,3 +7,43 @@ The [Reward Points]({% link marketing/rewards-loyalty.md %}) section of the cust
...
+
+1. Enter **Comments** for the points balance updating, if needed.
+
+    ![]({% link images/images-ee/reward-points-balance.png %}){: .zoom}
+    _Reward Points Balance_
+
+1. Also, the admin can subscribe a customer to **Reward Points Notifications**
+
+    - Subscribe for Balance Updates
+    - Subscribe for Points Expiration Notifications
```
Comment (leslietilling) [inline src/customers/account-dashboard-reward-points.md:34]:
```suggestion
1. Optionally, subscribe the customer to _Reward Points Notifications_:

    - **Subscribe for Balance Updates**
    - **Subscribe for Points Expiration Notifications**
```

File: src/customers/account-dashboard-reward-points.md [hunk 3]
```diff
@@ -7,3 +7,43 @@ The [Reward Points]({% link marketing/rewards-loyalty.md %}) section of the cust
...
+
+All actions related to reward points are displayed in the **Reward Points History** block.
+
+|Field|Description|
+|--- |--- |
+|Balance|Current balance of reward points for client|
+|Amount Balance|The amount of the current cash balance|
+|Points|The number of added or subtracted points|
+|Amount|Amount of money added or subtracted|
+|[Rate]({% link marketing/reward-exchange-rates.md %})|Reward exchange rate|
```
Comment (leslietilling) [inline src/customers/account-dashboard-reward-points.md:44]: Better to put the link in the description

```suggestion
|Rate|[Reward exchange rate]({% link marketing/reward-exchange-rates.md %})|
```

File: src/customers/account-dashboard-reward-points.md [hunk 4]
```diff
@@ -7,3 +7,43 @@ The [Reward Points]({% link marketing/rewards-loyalty.md %}) section of the cust
...
+|Website|Website to which the reward points history assigned|
```
Comment (leslietilling) [inline src/customers/account-dashboard-reward-points.md:45]:
```suggestion
|Website|Website to which the reward points history is assigned|
```

File: src/customers/account-dashboard-reward-points.md [hunk 5]
```diff
@@ -7,3 +7,43 @@ The [Reward Points]({% link marketing/rewards-loyalty.md %}) section of the cust
...
+|Reason|Reasons for awarding points:<br>**Making purchases** — every time you make a purchase you can earn points.<br>**Registering on the site** - accrued once upon registration.<br>**Subscribing to a newsletter** - accrued for the first time subscription.<br>**Sending Invitations** — Earn points by inviting your friends to join the site.<br>**Converting Invitations to Customer** — Earn points for every invitation you send out, leading to your friends registering on the site.<br>**Converting Invitations to Order** — Earn points for every invitation you send out which leads to a sale.<br>**Review Submission** — Earn points for submitting product reviews.|
```
Comment (leslietilling) [inline src/customers/account-dashboard-reward-points.md:46]:
```suggestion
|Reason|Reasons for awarding points:<br>**Making purchases** — Every time the customer makes a purchase they can earn points.<br>**Registering on the site** - Accrued upon registration (once).<br>**Subscribing to a newsletter** - Accrued for the first time subscription (once).<br>**Sending Invitations** — Earn points by inviting their friends to join the site.<br>**Converting Invitations to Customer** — Earn points for every invitation they send out, leading friends registering on the site.<br>**Converting Invitations to Order** — Earn points for each sale resulting from a sent invitation.<br>**Review Submission** — Earn points for submitting product reviews.|
```

**Review by leslietilling — CHANGES_REQUESTED** [review]
Some basic edits

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@leslietilling the pull request successfully imported.

**Pull request merged**